lib/faktory_worker/job.ex

```diff
@@ -218,7 +218,11 @@ defmodule FaktoryWorker.Job do
   defp push_pipeline_name(opts) do
     opts
     |> faktory_name()
-    |> FaktoryWorker.PushPipeline.format_pipeline_name()
+    |> format_pipeline_name()
+  end
+
+  defp format_pipeline_name(name) when is_atom(name) do
```

Comment:
Is this still needed?  Looks like the pipeline name may be able to be completely eliminated?